nautobot/docs/user-guide/administration/configuration/settings.md

```diff
@@ -156,11 +155,8 @@ in order for the repository to sync.
 ---
 
 ### `NAUTOBOT_LOG_DEPRECATION_WARNINGS`
+https://docs.nautobot.com/projects/core/en/latest/release-notes/version-1.5/?h=nautobot+v1.5#nautobot-apps-api-2723
```

Comment:
not sure I see how this addition is relevant?

Reply:
Good catch, I think it was a mistaken paste in the wrong window. Removing.